### ISSUE TITLE
a11y: search bar has no label

### ISSUE THREAD

**Issue by Charlotteis** [npm/newww#1797]
:wave: Hi team!

It is physically impossible for me not to do an a11y audit of websites I visit, so of course I ran this website through [a11y](https://www.npmjs.com/package/a11y).

There are a number of small fixable issues with the site (like contrasting colours for example) but one quick win is to fix the lack of label for the search bar!

```bash
🙊 :  ~charlotteis~  at 🖥  in ~/projects/
$ a11y https://www.npmjs.com/

  ✖ Controls and media elements should have labels

  #site-search
```

The way I tend to fix this in my own work is something like

```html
<div id="site-search-container">
  <label for='site-search' class='off-left'>search for a package</label>
  <input name="q" value="" type="search" id="site-search" placeholder="find packages" tabindex="1" autocorrect="off" autocapitalize="off" itemprop="query-input" autocomplete="off">
</div>
```

```css
.off-left {
    position: absolute;
    left: -999em;
}
```

^ the standard way of hiding elements from screen, but making sure they are still rendered on screenreaders or similar assistive technology.

I didn't want to jump in and fix it myself in case you have certain ways of writing your codes and I'd end up putting css/html in the wrong place, but let me know if I can help in any way if this is something that you deem fixable.

I looked at the a11y github label and searched for similar issues but it hadn't been mentioned that I could see, so I hope this helps!

Have a ~magical~ day and thanks for being you :sparkles:

**Comment by jefflembeck** [comment]
:+1: for whoever addresses this, we have `.a11y-only` as a class to add to this label.
